scripts/helpers/exec.js

```diff
@@ -0,0 +1,34 @@
+// Use concurrently for its great log handling and prefix capabilities
+
+const concurrently = require("concurrently");
+const ApplicationError = require("./applicationError");
+
+// even though I'm only ever running one thing at a time.
```

Comment:
I was a little confused by this comment. I think you're referring to using `concurrently` for prefixing/timestamping log output?

Reply:
Ah, looks like the ApplicationError require got inserted in-between the comment lines by my editor.